data/privacy_notices/privacy_notice_templates.yml

```diff
@@ -0,0 +1,227 @@
+privacy_notices:
+- name: Essential
+  description: This website uses cookies and other methods which are essential for you to use the website and its features.
+  internal_description: These cookies and services are essential for you to browse the website and use its features, such as accessing secure areas of the site. Cookies that allow web shops to hold your items in your cart while you are shopping online are an example of strictly necessary cookies. These cookies will generally be first-party session cookies. While it is not required to obtain consent for these cookies, what they do and why they are necessary should be explained to the user.
+  regions:
+    - eu_be
+    - eu_bg
+    - eu_cz
+    - eu_dk
+    - eu_de
+    - eu_ee
+    - eu_ie
+    - eu_el
+    - eu_es
+    - eu_fr
+    - eu_hr
+    - eu_it
+    - eu_cy
+    - eu_lv
+    - eu_lt
+    - eu_lu
+    - eu_hu
+    - eu_mt
+    - eu_nl
+    - eu_at
+    - eu_pl
+    - eu_pt
+    - eu_ro
+    - eu_si
+    - eu_sk
+    - eu_fi
+    - eu_se
+  consent_mechanism: notice_only
+  data_uses:
+    - provide
+  enforcement_level: not_applicable
+  disabled: False
+  has_gpc_flag: False
+  displayed_in_privacy_center: False
+  displayed_in_overlay: True
+  displayed_in_api: False
+- name: Analytics
+  description: This website uses cookies and services to collect information about how you use the website, like which pages you visited and which links you clicked on. None of this information can be used to identify you. It is all aggregated and, therefore, anonymized. Their sole purpose is to improve website functions. This includes cookies from third-party analytics services but are for the exclusive use of the owner of this website.
+  internal_description: These cookies and services collect information about how users use a website, like which pages they visit and which links they click on. None of this information can be used to identify visitors. It is all aggregated and, therefore, anonymized. Their sole purpose is to improve website functions. This includes cookies from third-party analytics services as long as the cookies and services are for your exclusive use.
+  regions:
+    - eu_be
+    - eu_bg
+    - eu_cz
+    - eu_dk
+    - eu_de
+    - eu_ee
+    - eu_ie
+    - eu_el
+    - eu_es
+    - eu_fr
+    - eu_hr
+    - eu_it
+    - eu_cy
+    - eu_lv
+    - eu_lt
+    - eu_lu
+    - eu_hu
+    - eu_mt
+    - eu_nl
+    - eu_at
+    - eu_pl
+    - eu_pt
+    - eu_ro
+    - eu_si
+    - eu_sk
+    - eu_fi
+    - eu_se
+  consent_mechanism: opt_in
+  data_uses:
+    - improve.system
+  enforcement_level: frontend
+  disabled: False
+  has_gpc_flag: False
+  displayed_in_privacy_center: False
+  displayed_in_overlay: True
+  displayed_in_api: False
```

Comment:
What kinds of notices get displayed in the API? Should this actually be true for everything?

Reply:
@mfbrown

Reply:
I'd say make this false for all the templates now. The API flag is meant to be for custom notices that customers create and expose/collect consent for in their own products and services. The expectation would be that all notices can be retrieved via an API and the API flag here is to support some future validation logic that allows a customer to create a notice for presentation in a different medium instead of the privacy center or overlay.

Reply:
Thanks for clarifying! Great, they're all False for now.